src/js/helpers.js

```diff
@@ -715,7 +715,7 @@ export default class Helpers {
   removeAllFields(stage) {
     const i18n = mi18n.current
     const opts = config.opts
-    const fields = stage.querySelectorAll(this.formBuilder.rowWrapperClassSelector)
+    const fields = stage.querySelectorAll(this.formBuilder.fieldSelector)
```

Comment:
fix is here and [here](https://github.com/kevinchappell/formBuilder/pull/1288/files#diff-0b7b9b2d8b0cee926f9118dd997d7d6fc80466c00f3a5a404041a3578b92ad14R65)

the rest is formatting changes

Reply:
this looks good but need to also check other places such as
openField
removeField
possibly prepareCloneWrappers

Reply:
just finished testing and all 3 worked for me but i did add a check if the row selector was present here https://github.com/kevinchappell/formBuilder/pull/1288/commits/b2eeae2b2aa0cbaf6bc6cff4d5bb6e04b78eeafc